docs/user-guide/troubleshooting.rst

```diff
@@ -61,3 +61,38 @@ install`` tool is able to verify the server's SSL certificate with a trusted CA.
 Details on how to install a new trusted CA are outside the scope of this
 document.
 
+Missing metadata.json file
+--------------------------
+
+Symptom
+^^^^^^^
```

Comment:
Remove this Symptom heading and convert this to paragraph form. The lines 64-65 will serve as the section heading for it.